### DISCUSSION TITLE
manage permissions

### DISCUSSION THREAD

**Discussion by jonatemps** [orchidsoftware/platform#1592] [Q&A]
Hello! 

How display my permissions in the screen?

please help me.

**Comment by tabuna** [comment]
Hi, please stop spamming. 
Registered permissions are automatically displayed on the user management page.

**Comment by noum** [comment]
@jonatemps `php artisan make:provider PermissionServiceProvider`
In app/Providers/PermissionServiceProvider.php, add the custom permissions.
In config/app.php's provider list, call PermissionServiceProvider class.

**Comment by jonatemps** [comment]
Thanks !
